BusBoard/BusBoard/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using RestSharp;
+using Newtonsoft.Json.Linq;
+
+namespace BusBoard
+{
+    internal class Program
+    {
+        public static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.Write("Enter a bus stop ID: ");
+                var stopId = Console.ReadLine();
+
+                var tflApiUrl = "https://api.tfl.gov.uk";
```

Comment:
I think you should move all of the TfL API logic out in to its own class to separate concerns